operation-external/src/main/java/org/sopt/makers/operation/client/eventbridge/EventBridgeImpl.java

```diff
@@ -0,0 +1,91 @@
+package org.sopt.makers.operation.client.eventbridge;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.annotation.PostConstruct;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import lombok.RequiredArgsConstructor;
+import org.sopt.makers.operation.client.eventbridge.dto.EventBridgeSenderRequest;
+import org.sopt.makers.operation.code.failure.AlarmFailureCode;
+import org.sopt.makers.operation.config.ValueConfig;
+import org.sopt.makers.operation.exception.AlarmException;
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.scheduler.SchedulerClient;
+import software.amazon.awssdk.services.scheduler.model.CreateScheduleRequest;
+import software.amazon.awssdk.services.scheduler.model.FlexibleTimeWindow;
+import software.amazon.awssdk.services.scheduler.model.FlexibleTimeWindowMode;
+import software.amazon.awssdk.services.scheduler.model.Target;
+
+@Component
+@RequiredArgsConstructor
+public class EventBridgeImpl implements EventBridgeSender {
+    private SchedulerClient schedulerClient;
+    private final ValueConfig valueConfig;
+    private ObjectMapper objectMapper;
```

Comment:
p2 

```suggestion
@Component
@RequiredArgsConstructor
public class EventBridgeImpl implements EventBridgeSender {

    private SchedulerClient schedulerClient;
    private final ValueConfig valueConfig;
    private ObjectMapper objectMapper;
```
첫 문장을 개행하는 컨밴션이 있습니다! 참고해주세요!

Reply:
네 ! 감사합니다 ! 신경쓰겠습니다 !!